tests/utils/BatchTestProcedures.sol

```diff
@@ -204,12 +209,31 @@ contract BatchTestProcedures is Test, DeployUtils, FfiUtils, DefaultMarketInput
     assertTrue(r.aToken != address(0), 'report.aToken');
     assertTrue(r.variableDebtToken != address(0), 'report.variableDebtToken');
     assertTrue(r.stableDebtToken != address(0), 'report.stableDebtToken');
-    assertTrue(r.emissionManager != address(0), 'report.emissionManager');
-    assertTrue(
-      r.rewardsControllerImplementation != address(0),
-      'r.rewardsControllerImplementation'
-    );
-    assertTrue(r.rewardsControllerProxy != address(0), 'report.rewardsControllerProxy');
+
+    if (config.incentivesProxy == address(0)) {
+      assertTrue(r.emissionManager != address(0), 'report.emissionManager');
+      assertTrue(
+        r.rewardsControllerImplementation != address(0),
+        'r.rewardsControllerImplementation'
+      );
+      assertTrue(r.rewardsControllerProxy != address(0), 'report.rewardsControllerProxy');
+    } else {
```

Comment:
Some of the assert inside here are duplicated (could be common for both logic branches), no?

Reply:
true, done!